superset-frontend/spec/javascripts/dashboard/components/Header_spec.jsx

```diff
@@ -83,7 +84,11 @@ describe('Header', () => {
   };
 
   function setup(overrideProps) {
-    const wrapper = shallow(<Header {...props} {...overrideProps} />);
+    const wrapper = mount(
+      <ThemeProvider theme={supersetTheme}>
```

Comment:
You can skip this, and the `superset-ui` imports, by using the helper util:
`import { styledMount as mount } from 'spec/helpers/theming';`
or 
`import { styledShallow as shallow } from 'spec/helpers/theming';`

Reply:
Fixed 😉